Jenkinsfile_CNP

```diff
@@ -37,6 +39,8 @@ withPipeline(type, product, component) {
     disableLegacyDeployment()
     loadVaultSecrets(secrets)
 
+    env.POSTGRES_HOST = "wa-task-management-api-postgres-db-aat.postgres.database.azure.com"
```

Comment:
see my comment on https://github.com/hmcts/wa-task-management-api/pull/267/files about the name of the DB.

Reply:
Thanks will apply change